mwapi/errors.py

```diff
@@ -36,3 +48,38 @@ class LoginError(RuntimeError):
     @classmethod
     def from_doc(cls, doc):
         return cls(doc.get('result'))
+
+
+class RequestError(requests.exceptions.RequestException):
```

Comment:
I want to be able to handle different types and it's awkward in our docs to say: "You can't catch exceptions like normal.  You're going to need to catch our special exception and then use isinstance() to check it against `requests`"  Also it would mean that we could never switch from requests.